source/stylesheets/components/_newsletter.scss

```diff
@@ -0,0 +1,71 @@
+// Newsletter
+//
+// A custom grid for the newsletter section.
+//
+// markup:
+// <div class="Newsletter">
+//   <div class="Newsletter-input">
+//     <div class="InputText">
+//       <label class="InputText-label">What's your email</label>
+//       <input class="InputText-input" type="email"></input>
+//     </div>
+//   </div>
+//   <div class="Newsletter-submit">
+//     <input class="ButtonPrimary"></input>
+//   </div>
+// </div>
+
+.Newsletter {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-between;
+  max-width: 600px;
+  margin: auto;
+}
+
+.Newsletter-inputWrapper {
+  flex: 2 0 auto;
+}
+
+.Newsletter-input {
+  @include font-properties(heading, small);
+  width: 100%;
+
+  padding: $Theme-spacing-xSmall $Theme-spacing-small;
+
+  border: 1px solid $Theme-color-poloBlue;
+  border-radius: 4px;
+
+  color: $Theme-color-horizon;
+}
+
+@include media('>=tablet') {
+  .Newsletter-inputWrapper {
+    margin-right: $Theme-spacing-small;
+  }
+}
+
+@include media('<tablet') {
+  .Newsletter {
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .Newsletter-inputWrapper {
+    width: 100%;
+    padding: 0 $Theme-spacing-small;
+  }
+
+  .Newsletter-submit {
+    width: 100%;
+    padding: $Theme-spacing-small;
+  }
+}
+
+.Newsletter {
+  ::placeholder {
+    color: $Theme-color-poloBlue;
+    opacity: 1;
```

Comment:
I'm guessing this is a design decision, but why do we want a placeholder to be opaque? Doesn't it look like the input is already filled?